lib/main.dart

```diff
@@ -230,12 +230,12 @@ Future<String> findLatestSdkForRelease(String sdkRelease) async {
   final http = HttpClient(
     'setup-dart',
     <JSAny>[].toJS,
-    jsify({
+    {
       'allowRedirects': true,
       'maxRedirects': 3,
       'allowRetries': true,
       'maxRetries': 3,
-    }) as JSObject?,
+    }.toJSBox,
```

Comment:
@srujzs - also, I don't see a `.toJS` on Dart Maps, but do see a `toJSBox`. Here, I'm trying to convert to a regular JSObject.

Reply:
I'd just use `.jsify()` like you were before - it's just an extension method now. This should translate the map into a JS object using the string keys as property names just like before. `toJSBox` creates an explicit box around the object so that you can pass it back to the Dart runtime later so it's not the right fit.

Reply:
👍  thanks